registry/server/index.ts

```diff
@@ -1,16 +1,39 @@
 import path from 'path';
 
+['log', 'warn', 'error'].forEach(function (method) {
+    //@ts-ignore
+    var old = console[method];
+    //@ts-ignore
+    console[method] = function () {
+        //@ts-ignore
+        var stack = new Error().stack.split(/\n/);
+        // Chrome includes a single "Error" line, FF doesn't.
+        if (stack[0].indexOf('Error') === 0) {
+            stack = stack.slice(1);
+        }
+        //@ts-ignore
+        var args = [].slice.apply(arguments).concat([stack[1].trim()]);
+        return old.apply(console, args);
+    };
+});
+import app from './app';
+import { loadPlugins } from './util/pluginManager';
+import server from './server';
+import { getLogger } from './util/logger';
+
 process.env.NODE_CONFIG_DIR = path.resolve(__dirname, '../config');
 require('newrelic'); //Should be lower than NODE_CONFIG_DIR env var definition
 
-import app from './app';
-import server from './server';
+let log = console.error;
 
 (async () => {
     try {
+        loadPlugins();
```

Comment:
I see that `loadPlugins()` is called 2 times here and inside `app()`. Is it done intentionally?

Reply:
fixed